code/control/DNRoot.php

```diff
@@ -823,6 +852,28 @@ public function DNProjectList() {
 	}
 
 	/**
+	 * Provide a list of all projects.
+	 *
+	 * @return SS_List
+	 */
+	public function getStarredProjects() {
+		$member = Member::currentUser();
+		if(!$member) {
```

Comment:
Should this check be `$member === null` like in the `starproject` controller action?